src/Select.js

```diff
@@ -59,10 +64,17 @@ class Select extends React.Component {
 			'onOptionRef',
 			'removeValue',
 			'selectValue',
+			'setAutofill',
+			'clearAutofill',
+			'setAutofilled',
+			'clearAutofilled',
 		].forEach((fn) => this[fn] = this[fn].bind(this));
 
+		this.autoCompleteEnabled = !this.props.multi && this.props.autoComplete;
```

Comment:
This looks like it should be a method that get's called where needed, since these props could change and the constructor runs only once.